core/templates/dev/head/pages/profile/profile.html

```diff
@@ -134,12 +134,13 @@ <h2 class="oppia-profile-username-small-screen">
       </md-content>
 
       <span class="oppia-profile-portfolio-pages" ng-if="numUserPortfolioExplorations > 6">
-        <i class="material-icons md-18" ng-disabled="currentPageNumber == 0"
+        <i ng-class="{'material-icons md-18 oppia-navigation-button':true,
+                      'disabled':startingExplorationNumber==1}"
            ng-click="goToPreviousPage()">
           &#xE5C4;
         </i>
-        Showing <[currentPageNumber * PAGE_SIZE + 1]> - <[Math.min(numUserPortfolioExplorations, (currentPageNumber + 1) * PAGE_SIZE)]> of <[numUserPortfolioExplorations]>
-        <i class="material-icons md-18" ng-disabled="(currentPageNumber + 1) * PAGE_SIZE >= userEditedExplorations.length" ng-click="goToNextPage()">&#xE5C8;</i>
+        Showing <[startingExplorationNumber]> - <[endingExplorationNumber]> of <[numUserPortfolioExplorations]>
+        <i ng-class="{'material-icons md-18 oppia-navigation-button':true,'disabled':endingExplorationNumber >= numUserPortfolioExplorations}"  ng-click="goToNextPage()">&#xE5C8;</i>
       </span>
```

Comment:
It is getting quite crowded up here it seems like.
can you put some spacing between the dict key and value and also right after each `,`:
```javascript
{'material-icons md-18 oppia-navigation-button': true, 'disabled': endingExplorationNumber >= numUserPortfolioExplorations}"
```

```

Reply:
Also Sean suggested you to use `track by $index`, take a look at how `exploration-summary-tile` uses ng-repeat and other AngularJS 1.x docs and make this cleaner please.

You have the right idea and logic here, just need to make it more "Angularly"

Reply:
done..!